src/handlers/gestures/GestureDetector/Wrap.web.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef } from 'react';
+import type { LegacyRef } from 'react';
+import { tagMessage } from '../../../utils';
+
+let wrapID = 0;
+
+export const Wrap = forwardRef(
+  ({ children }: { children?: React.ReactNode }, ref) => {
```

Comment:
Can the type be set as generic type of `forwardRef`? I think it would look something like `forwardRef<HTMLElement, PropsWithChildren<{}>`

Reply:
Sure, done in [4fab7ef](https://github.com/software-mansion/react-native-gesture-handler/pull/3127/commits/4fab7eff03080207fea1f8cfcc904958de50fade)